### PR TITLE
fixs.

### PR DESCRIPTION
**Pull request by Argalep**
* Cleanup Reward MonsterStatuette  fix.
* Veteran Reward MonsterStatuette fix.

### CODE REVIEW

**Comment by kevin-10** [comment]
Why the params in CollectionItem?

File: Scripts/Items/Decorative/MonsterStatuette.cs [hunk 1]
```diff
@@ -184,6 +171,8 @@ public class MonsterStatuette : Item, IRewardItem
         private MonsterStatuetteType m_Type;
         private bool m_TurnedOn;
         private bool m_IsRewardItem;
+        private Timer m_NewsTimer;
```
Comment (kevin-10) [inline Scripts/Items/Decorative/MonsterStatuette.cs]: So, if we're using the old Sherry the Mouse script, while are we adding Town Crier functionality to MonsterStatuette?

Comment (Argalep) [inline Scripts/Items/Decorative/MonsterStatuette.cs]: I could not decide.  It seems more accurate to use the code in MonsterStatuette

Comment (kevin-10) [inline Scripts/Items/Decorative/MonsterStatuette.cs]: Ther is no need for the town crier functionality in MonsterStatuette.

**Pull request merged**